packages/grid/src/CellRenderer.ts

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable class-methods-use-this */
+import { getOrThrow } from '@deephaven/utils';
+import { isExpandableGridModel } from './ExpandableGridModel';
+import { VisibleIndex, Coordinate, BoxCoordinates } from './GridMetrics';
+import { GridRenderState } from './GridRenderer';
```

Comment:
I recall the motivation for why I wanted to export all the grid types to their own files, but that may have been a bit extreme for what we need here.
The circular import is here, CellRenderer imports GridRenderer, which also imports the CellRenderer type.
Instead, move `EditingCell` and `GridRenderState` out to a `GridRendererTypes.ts` file, and just import `GridRenderState` from there for both, then you no longer have a circular dependency.